src/modules/preview-site/components/create-preview-button.tsx

```diff
@@ -74,6 +95,8 @@ export function CreatePreviewButton( { onClick, selectedSite }: CreatePreviewBut
 		tooltipContent = { text: errorMessages.rest_site_creation_blocked };
 	} else if ( isOverLimit ) {
 		tooltipContent = { text: overLimitMessage };
+	} else if ( ! isPhpVersionDefault || isWpVersionBelowDefault ) {
```

Comment:
Not critical, but the word "Below" looks weird a bit 😅
In the context of code you most likelly wanted to menthion the below in the 44-47 lines, but I think it's redundant and a bit confusing.
```suggestion
	} else if ( ! isPhpVersionDefault || isWpVersionDefault ) {
```

Reply:
Hm, also, is it correct condition `isWpVersionBelowDefault`? Shouldn't it be `! isWpVersionBelowDefault`?